data/ch10/ch10_load.sh

```diff
@@ -40,11 +40,11 @@ echo "Extracting data files for loading..."
 tar xvzf movies_dev_data.tar.gz
 
 echo "Loading vertices into the graph $DEST_KS"
-$DSBULK load -k $DEST_KS -t Movie -url Movie.csv -header true
```

Comment:
maybe rename `DEST_KS` into `DEST_GRAPH` ?

Reply:
@alexott yes, that is a great suggestion for a new variable name. I will add that to a different issue.